test/Common.hs

```diff
@@ -38,6 +40,14 @@ import Test.QuickCheck.Instances            as T ()
 
 import qualified Data.Tree.AVL as AVL
 
+instance Binary x => AVL.Serialisable x where
```

Comment:
So the PR doesn't remove `binary` dependency and it moves to its own serialisable class only partially. What's the deal?

```
ξ> git grep Binary
test/Common.hs:import Data.Binary                               (Binary, encode, decodeOrFail)
test/Common.hs:instance Binary x => AVL.Serialisable x where
test/Common.hs:instance Binary Layer
test/Common.hs:instance Binary   AVL.Tilt
test/Common.hs:    deriving (Eq, Ord, Generic, Binary)
````